package.json

```diff
@@ -9,7 +9,7 @@
     "url": "git://github.com/cdnjs/cdnjs.git"
   },
   "scripts": {
-    "test": "vows"
+    "test": "vows && jasmine"
```

Comment:
I wonder if we could separate this part of test to something like `devTest` as we use `npm test` for libraries' package.json validation for a long time, what do you think?